CMakeLists.txt

```diff
@@ -201,6 +202,11 @@ if(BUILD_BENCH)
   add_definitions(-DSUPP_BENCH)
 endif()
 
+if (NNG_ENABLE_PARQUET)
```

Comment:
If `ENABLE_PARQUET` should be detect and set `NNG_ENABLE_PARQUET`.

Reply:
Good caught. fixed